sphinx_gallery/gen_rst.py

```diff
@@ -284,34 +287,19 @@ def get_md5sum(src_file):
     return src_md5
 
 
-def check_md5sum_change(src_file):
-    """Returns True if src_file has a different md5sum"""
+def md5sum_is_current(src_file):
```

Comment:
It doesn't seem like the md5 check is working as expected. When I do `make html` it looks like all the examples are rerun if I trust the output:

```
plotting code blocks in ../examples/plot_choose_thumbnail.py
 - time elapsed : 0.14 sec
plotting code blocks in ../examples/plot_exp.py
 - time elapsed : 0.14 sec
plotting code blocks in ../examples/plot_function_identifier.py
 - time elapsed : 0.048 sec
plotting code blocks in ../examples/plot_gallery_version.py
 - time elapsed : 0.089 sec
plotting code blocks in ../examples/plot_quantum.py
 - time elapsed : 0.089 sec
plotting code blocks in ../examples/plot_raise.py
 - time elapsed : 0.073 sec
plotting code blocks in ../examples/plot_raise.py
.
.
.
```
```